linux_os/guide/system/software/integrity/crypto/configure_libreswan_crypto_policy/rule.yml

```diff
@@ -26,6 +26,10 @@ ocil_clause: |-
     <tt>/etc/ipsec.conf</tt> does not contain <tt>include /etc/crypto-policies/back-ends/libreswan.config</tt>
 
 ocil: |-
+    Either Libreswan is not installed, or it is configured to use the system-wide defined crypto policy.
+
+    {{{ ocil_package(package="libreswan") }}}
+
```

Comment:
This should be in the OCIL clause and `{{{ ocil_package(package="libreswan") }}}` is not needed.